docs/medical-api/getting-started/quickstart.mdx

```diff
@@ -215,21 +215,20 @@ See the [Get Document endpoint](/medical-api/api-reference/document/get-document
 
 - In the side panel that comes up after clicking `Documents`, simply click `Download` on the desired Document.
 
-That's it! You're now well equiped to use the Metriport API to access and manage your Patients' medical data.
+## 8. Get Patient FHIR Resources 🎉🎉🎉
```

Comment:
I still don't see the overview updated?

Reply:
fter the conversion from CDA to FHIR, we will store the resulting FHIR resources and make them available to be queried. This will allow you to get the necessary data you might need to better understand and treat your Patients.This part ?

Reply:
this part https://github.com/metriport/metriport/blob/8d0c02bab18fe984dfcdca9afeb1b77d1fe643db/docs/medical-api/getting-started/quickstart.mdx#L20-L26